src/routes/common/pagination/pagination.data.ts

```diff
@@ -3,7 +3,7 @@ const QUERY_PARAM_LIMIT = 'limit';
 const QUERY_PARAM_OFFSET = 'offset';
 
 export class PaginationData {
-  private constructor(readonly limit?: number, readonly offset?: number) {}
+  constructor(readonly limit?: number, readonly offset?: number) {}
```

Comment:
I'm setting this constructor as public since I'm using it to create adjusted paginations [here](https://github.com/5afe/safe-client-gateway-nest/pull/215/files#diff-45bd14f413961db261e493c23aa045bdf6f69f6103f6fc34122688642e8116acR238). I can't see any downside on this, but another option would be using a builder pattern and keep the constructor private, feel free to comment 🙂